ctf.schema.json

```diff
@@ -0,0 +1,193 @@
+{
+  "$id": "JsonConfiguration",
+  "type": "object",
+  "properties": {
+    "title": {
+      "description": "The title of the event.",
+      "type": "string"
+    },
+    "flagValidityInRounds": {
+      "description": "Validity of a flag in rounds.",
+      "type": "integer",
+      "minimum": 0.0,
+      "maximum": 9.223372036854776E+18
+    },
+    "checkedRoundsPerRound": {
+      "description": "Number of times a flag is checked per round.",
+      "type": "integer",
+      "minimum": 1.0,
+      "maximum": 9.223372036854776E+18
+    },
+    "roundLengthInSeconds": {
+      "description": "The length of one round in seconds.",
+      "type": "integer",
+      "minimum": 1.0,
+      "maximum": 9.223372036854776E+18
```

Comment:
Is this schema autogenerated now? Else we could do more useful maxima than "21x age of the universe" seconds :D
Obviously not a big deal for now, just wondering :)

Reply:
Yes it's auto-generated. I made the core package executable for that purpose.

Reply:
But you can put in any number you want in the .cs file describing the entity.